src/stackedchart/StackedChart.tsx

```diff
@@ -101,10 +127,13 @@ export default class StackedChart extends React.Component<
         const data = await getStackedChart({
             contact: this.props.contact,
             type: this.props.type,
+            fipsKeyForState: this.props.fipsKeyForState,
         });
-        if (data != null) {
+        if (data != null && data[0] != null && data[0].xAxisData.length > 0) {
             const options = optionsDeletgate(toHighChartData(data[0]));
             this.setState({ options });
+        } else {
+            this.setState({ displayNoDataMessage: true });
```

Comment:
Is this possible? I think all the states must have data

Reply:
yes i checked.

Reply:
should i remove this logic or leave it handle if it occurs in the future.